controllers/node_controller.go

```diff
@@ -122,22 +121,9 @@ func (r *NodeReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, nil
 	}
 
-	cluster, err := r.getClusterForNode(node)
+	labels, err := r.fallbackIdentifier.IdentifyNode(ctx, node)
```

Comment:
What does fallback Identifier use to identify the node?

Reply:
ec2 instancef tag